client/src/components/Header/components/menu-button.tsx

```diff
@@ -2,12 +2,14 @@ import React, { RefObject } from 'react';
 import { useTranslation } from 'react-i18next';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBars } from '@fortawesome/free-solid-svg-icons';
+import { User } from '../../../redux/prop-types';
 interface MenuButtonProps {
```

Comment:
Just minor formatting

```suggestion
import { User } from '../../../redux/prop-types';

interface MenuButtonProps {
```